Aptoide/aptoide-client-v8
# [Hotfix/ASV-2019] File not found error

**What does this PR do?**

   This PR aims at quick fixing the download file not found error handle. Before, if there was an error on the download on the 1st path, it would retry for the alternative one. The problem is that if this alternative one also failed, then it would be on a loop retrying this failed alternative link.

Note: Also took the chance to removed an unused class.

**Database changed?**

   No

**Where should the reviewer start?**

- [ ] RetryFileDownloadManager.java

**How should this be manually tested?**

  Do a rewrite where you break the path links so that it returns 404 or use the rewrite I added on the ticket. Check that after the download starts it will return a general error.

**What are the relevant tickets?**

  Tickets related to this pull-request: [ASV-2019](https://aptoide.atlassian.net/browse/ASV-2019)

**Questions:**

   Does this add new dependencies which need to be added? (Eg. new keys, etc.) 



**Code Review Checklist**
